setup/stability-extra/multicluster-vpn/locality-failover/setup.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
```

Comment:
Failover won't work without an outlierDetection rule like
```
apiVersion: networking.istio.io/v1alpha3
kind: DestinationRule
metadata:
  name: outlier
spec:
  host: "google.com"
  trafficPolicy:
    outlierDetection:
      consecutiveErrors: 7
      interval: 5m
      baseEjectionTime: 15m
```

(It can be whatever, just outlierDetection must exist)